core/services/fluxmonitorv2/orm.go

```diff
@@ -93,25 +93,26 @@ func (o *orm) CountFluxMonitorRoundStats() (int, error) {
 
 // CreateEthTransaction creates an ethereum transaction for the BPTXM to pick up
 func (o *orm) CreateEthTransaction(
+	db *gorm.DB,
 	fromAddress common.Address,
 	toAddress common.Address,
 	payload []byte,
 	gasLimit uint64,
 	maxUnconfirmedTransactions uint64,
 ) error {
-	db, err := o.db.DB()
+	sqlDB, err := db.DB()
 	if err != nil {
 		return errors.Wrap(err, "orm#CreateEthTransaction")
 	}
 
-	err = utils.CheckOKToTransmit(context.Background(), db, fromAddress, maxUnconfirmedTransactions)
+	err = utils.CheckOKToTransmit(context.Background(), sqlDB, fromAddress, maxUnconfirmedTransactions)
 	if err != nil {
 		return errors.Wrap(err, "orm#CreateEthTransaction")
 	}
 
 	value := 0
 
-	dbtx := o.db.Exec(`
+	dbtx := db.Exec(`
```

Comment:
This exact (or close to exact) code is spread around at least three places in the codebase. Can we abstract into one method in bulletprooftxmanager?

Reply:
yeah @RyanRHall has a ticket for exactly that already